attrs_strict/_error.py

```diff
@@ -89,3 +90,17 @@ def __str__(self):
         )
 
         return self._render(error)
+
+
+def _is_newtype(type_):
+    """Checks if `type` is :func:`typing.NewType`."""
+    return (hasattr(type_, "__qualname__") and
+            type_.__qualname__ == "NewType.<locals>.new_type")
+
+
+def _format_type(type_):
+    """Returns a string to pretty display a given `type_` to the user."""
```

Comment:
I think this function is self explanatory, I'm not sure if this is needed. Sorry I should have noticed this previously.

Reply:
No necessarily, you can format a object to a string for multiple purposes, here it described that it should be formatted for short user display.
Furthermore, It is also a common best practice to always document functions – linter or code quality tools usually enforce a docstring in every function and class.